From bf6326285460b56d0eb815af8d0e51815cf840a1 Mon Sep 17 00:00:00 2001
From: Nicolin Chen <Guangyu.Chen@freescale.com>
Date: Wed, 18 Jun 2014 17:46:49 +0800
Subject: [PATCH 0414/1074] ENGR00318773-11 ARM: imx6qdl: Remove incorrect
 clock sources

The clock sources from ASRC, ESAI and MLB are not actually from CCM but
outside PAD inputs. So we these clock sources over here are totally wrong
and should be fixed.

Acked-by: Wang Shengjiu <b02247@freescale.com>
Signed-off-by: Nicolin Chen <Guangyu.Chen@freescale.com>
[Original patch taken from git://git.freescale.com/imx/linux-2.6-imx.git]
Signed-off-by: Biyao Zhai <biyao.zhai@windriver.com>
---
 arch/arm/boot/dts/imx6qdl.dtsi |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/arch/arm/boot/dts/imx6qdl.dtsi b/arch/arm/boot/dts/imx6qdl.dtsi
index 9069793..84ca33a 100644
--- a/arch/arm/boot/dts/imx6qdl.dtsi
+++ b/arch/arm/boot/dts/imx6qdl.dtsi
@@ -175,9 +175,9 @@
 					       <&sdma 15 18 0>;
 					dma-names = "rx", "tx";
 					clocks = <&clks 197>, <&clks 3>,
-						 <&clks 197>, <&clks 107>,
-						 <&clks 0>,   <&clks 118>,
-						 <&clks 0>,  <&clks 139>,
+						 <&clks 197>, <&clks 0>,
+						 <&clks 0>,   <&clks 0>,
+						 <&clks 62>,  <&clks 0>,
 						 <&clks 0>;
 					clock-names = "core",  "rxtx0",
 						      "rxtx1", "rxtx2",
-- 
1.7.5.4

